### ISSUE TITLE
project manager can review start pack info and documents

### ISSUE THREAD

**Issue by JMurphyWeb** [karmaradio/karma#17]
As a project manager,
I can view the startpack info and relating documents belonging to a contractor that has 'accepted' an offer.

**Comment by JMurphyWeb** [comment]
@jonscottfilm what needs to be done with this information? Is it enough for the PM to be able to view the required fields along with any supporting documents?

**Comment by jonscottfilm** [comment]
yes, view-only, but for selected authorised project users.

any further distribution of this data, for mvp, can be handled by webmerge. 
eg. payroll data to accountant, in custom form for sending on to payroll company, and also in shared spreadsheet.

**Comment by JMurphyWeb** [comment]
Sorry I don't understand. Do they need to be able to download the data?

**Comment by jonscottfilm** [comment]
Non-mvp features will enable authorised project users to use this data for further tasks. In the meantime, until we have a fully functional viewer of all data in all user profiles, and the ability to view and export custom reports, then Webmerge can deliver the required data to the few project users who need it. Eg, the Payroll Accountant can view the bank details of a user who's recently submitted their start pack, to prepare that user on the coming payroll.

If easy to build, downloadable data is ok, and obviously easier to copy into other documents, but downloading then creates multiple, offline versions.

Maybe something easier to discuss in person.

**Comment by JMurphyWeb** [comment]
@jonscottfilm it is slightly unclear what needs to be done here for MVP. Does the PM need to be able to view this data? Its worth thinking about priority for this one.

Here's how I had imagined it would work:
Contractor accepts offer -> then from the offer dashboard, PM can view that contractor, and any details/settings from their personal settings that will be needed to form the contract.

**Comment by JMurphyWeb** [comment]
@jonscottfilm as agreed removing from milestone 2

**Comment by JMurphyWeb** [comment]
Removing from milestone 4 as #405  covers the supporting documents

**Comment by jonscottfilm** [comment]
This will be covered by Export, Phase 2.
